src/lib/components/compositions/radio/animated.svelte

```diff
@@ -5,15 +5,16 @@
   import type { Maybe } from '$lib/interfaces';
   import Label from '$components/primitives/label/label.svelte';
 
-  export let defaultOption: number;
+  export let defaultOption: string | null = null;
```

Comment:
Can we use the `Maybe` here for consistency? So, `Maybe<string>`, rather than `string | null`?